### DISCUSSION TITLE
Timer not showing on Firefox Android

### DISCUSSION THREAD

**Discussion by cattailtip** [proginosko/LeechBlockNG#444] [Q&A] [answered]
Hi, I'm running into an issue where the countdown timer on a page is not showing in the corner when using Leechblock on the Firefox Android app. After searching the docs and this Q&A, I went to about:addons and checked the Leechblock extension permissions, but I don't see the permission called "Access your data for all websites" anywhere, which seemed to be the root cause for timers not showing in other cases (I do see it on desktop though).

I'd been using Leechblock with no issues on an old phone, and when I had to switch I used the export/import option to set up Leechblock on my new phone (Samsung) so all the settings would be the same. On the old phone, I noticed the same set of permissions (sans "Access your data for all websites"), but the timer would show on blocked webpages on the old phone.

Is there some setting or step I'm missing to enable the permission/timer? I've tried removing & re-adding the extension, as well as re-installing Firefox, with no luck. The extension still blocks fine otherwise, but the timer was a real nice thing to have for a very helpful extension. Thanks.

**Comment by proginosko** [comment, marked as answer]
Yes, this is a known issue, unfortunately due to a [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1872600) in Firefox for Android. If you open the LeechBlock menu, the timer will probably appear, but that's not a very convenient workaround. 🙁

**Reply by cattailtip** [reply]
Ah I see, that's unfortunate! Thank you for the response.

**Comment by bhavin192** [comment]
The bug is fixed in Firefox Android v128, I had to remove the add-on and install it again to get the timer working :tada:

**Reply by proginosko** [reply]
That's good to hear!

**Reply by cattailtip** [reply]
Great to hear! Thank you for replying, I was able to pick up the update and get the timer working for myself as well
